test_conformance/c11_atomics/test_atomics.cpp

```diff
@@ -366,11 +388,25 @@ class CBasicTestLoad
         correct = true;
         for (cl_uint i = 0; i < threadCount; i++)
         {
-            if (refValues[i] != (HostDataType)i)
+            if (std::is_same<HostDataType, cl_half>::value)
```

Comment:
You may need to pull in the latest changes that enable C++ 17 compilation also.

```suggestion
            if constexpr (std::is_same<HostDataType, cl_half>::value)
```

Reply:
This is great :)